internal/store/label.go

```diff
@@ -0,0 +1,57 @@
+package store
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+	"github.com/kubev2v/migration-planner/internal/store/model"
+	"github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+)
+
+type Label interface {
```

Comment:
Unless you want to create a separate endpoint to update labels independently for a source, you don't need this.
Gorm can handle labels in the source store.